src/server/game/Mails/Mail.cpp

```diff
@@ -211,8 +211,8 @@ void MailDraft::SendMailTo(SQLTransaction& trans, MailReceiver const& receiver,
     stmt->setUInt16(++index, GetMailTemplateId());
     stmt->setUInt32(++index, sender.GetSenderId());
     stmt->setUInt32(++index, receiver.GetPlayerGUIDLow());
-    stmt->setString(++index, GetSubject());
```

Comment:
I think these 2 strings should be kept non null. There's no reason for empty strings to be nulls